bubble/Podfile

```diff
@@ -12,6 +12,7 @@ target 'bubble' do
   pod "FacebookCore"
   pod "FacebookLogin"
   pod "GoogleSignIn"
+  pod "ParseFacebookUtilsV4"
```

Comment:
why do we have Parse cocoapod? remove it and commit if it's not needed

Reply:
I  have a need for it just not using it yet, let me take it out when am certainly am not gonna use it